propel/schema.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<database name="pizzaService">
+    <table name="ingredients" phpName="Ingredient">
+        <column name="id" type="integer" required="true" primaryKey="true" autoIncrement="true" />
+        <column name="name" type="varchar" size="120" required="true" />
+    </table>
+
+    <table name="pizzas" phpName="Pizza">
+        <column name="id" type="integer" required="true" primaryKey="true" autoIncrement="true" />
+        <column name="name" type="varchar" size="120" required="true"/>
+        <column name="price" type="float" required="true"/>
+    </table>
+
+    <table name="pizza_ingredients" phpName="Ingredient_pizza" isCrossRef="true">
```

Comment:
Please change the phpName to `IngredientPizza`. This will cause that new propel model classes are generated, but you must delete the old generated classes on your own. (`Ingredient_pizza.php`,`Ingredient_pizzaPeer.php`, `Ingredient_pizzaQuery.php` and the corresponding classes in the `om/` and `map/` directories.